packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -215,6 +215,11 @@ export default {
             default: ''
         },
 
+        otacToAuthExchanger: {
+            type: Function,
+            default: () => ''
```

Comment:
Suggestion: Could log a not implemented message

Reply:
Logger would not be available here as per: https://vuejs.org/v2/guide/components-props.html
"Note that props are validated before a component instance is created, so instance properties (e.g. data, computed, etc) will not be available inside default or validator functions."

Reply:
@tommcclean I replaced it with default implementation that throws not implemented exception.